tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/Configs/ConfigRepository.cs

```diff
@@ -85,6 +93,94 @@ public List<DynamicTableEntity> GetConfigEntities(string filter)
             return entities;
         }
 
-    }
-}
 
+        /// <summary>
+        /// Update  or Insert Configuration values 
+        /// </summary>
+        /// <param name="configs"></param>
+        public bool UpdateEntityProperty(string configs)
+        {
+            try
+            {
+                CloudStorageAccount cloudStorageAccount = CloudStorageAccount.Parse(generalSettings.CloudStorageConnectionString);

```

Comment:
These lines of code might be present in all the methods/api. Try to write some helper class or method to get this information, so that code might not be repetitive